### PR TITLE
  Add verbosity level to show each part of list command

### PR DESCRIPTION
**Pull request by steevanb**
To solve https://github.com/Huttopia/console-bundle/issues/13.

### CODE REVIEW

File: Descriptor/TextDescriptor.php [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        int $availableCommandsVerbosityLevel
+    ) {
+        $this->verbosity = $verbosity;
+        $this->symfonyVersionVerbosityLevel = $symfonyVersionVerbosityLevel;
+        $this->usageVerbosityLevel = $usageVerbosityLevel;
+        $this->optionsVerbosityLevel = $optionsVerbosityLevel;
+        $this->availableCommandsVerbosityLevel = $availableCommandsVerbosityLevel;
+    }
+
+    protected function describeApplication(Application $application, array $options = [])
```
Comment (steevanb) [inline Descriptor/TextDescriptor.php:47]: I can't add a return type cause it's `string|mixed` (WTF?) as Symfony PHPDoc said

File: Descriptor/TextDescriptor.php [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    {
+        if (
+            $this->symfonyVersionVerbosityLevel === 0
+            && $this->usageVerbosityLevel === 0
+            && $this->optionsVerbosityLevel === 0
+            && $this->availableCommandsVerbosityLevel === 0
+        ) {
+            return parent::describeApplication($application, $options);
```
Comment (steevanb) [inline Descriptor/TextDescriptor.php:55]: As Symfony don't use `protected` for internal code, and everything is hard coded in `describeApplication()`, i add this test to call parent method if no configuration will change the output.

With it, this override is compatible with any Symfony version if no specific configuration is made.

**Comment by ZeMarine** [comment]
There is a conflict. Thanks for the contrib :)

**Comment by steevanb** [comment]
The conflict is fixed :)

**Review by ZeMarine — APPROVED** [review]

**Pull request merged**